aspnetcore/performance/caching/middleware.md

```diff
@@ -1,15 +1,104 @@
 ---
-title: 🔧 Response caching middleware
+title: Response Caching Middleware | Microsoft Docs
 author: rick-anderson
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 12/05/2016
 ms.topic: article
 ms.assetid: f9267eab-2762-42ac-1638-4a25d2c9d67c
 ms.prod: aspnet-core
 uid: performance/caching/middleware
 ---
-# 🔧 Response caching middleware
+# Response Caching Middleware
+
+By [Luke Latham](https://github.com/GuardRex) and [John Luo](https://github.com/JunTaoLuo)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/response/sample)
+
+This document provides details on how to configure the Response Caching Middleware for ASP.NET Core applications. For an introduction to to HTTP caching and the `ResponseCaching` feature, see [Response Caching](response.md).
```

Comment:
I'm changing it to `ResponseCacheAttribute`, but I'm confused looking at `response.md` doc and the attribute itself ...

Matches the `response.md` doc text, so I'll use it in this doc ...
> ... and the `ResponseCacheAttribute`, see ...

Matches the `response.md` section header ...
> ... and the `ResponseCache Attribute`, see ...

Matches the name of the attribute (but doesn't appear in the `response.md` doc ...
> ... and the `ResponseCache` attribute`, see ..."